api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -9,41 +9,6 @@
  */
 package org.openmrs.api;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsInAnyOrder;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.empty;
-import static org.hamcrest.Matchers.hasItems;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.nullValue;
-import static org.openmrs.Order.Action.DISCONTINUE;
-import static org.openmrs.Order.FulfillerStatus.COMPLETED;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.openmrs.test.OpenmrsMatchers.hasId;
-import static org.openmrs.test.TestUtil.containsId;
-
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import java.lang.reflect.Field;
-import java.text.DateFormat;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Locale;
-import java.util.Set;
-
```

Comment:
@tendomart are these formatting changes in the imports needed ?

Reply:
@gitcliff have you had a look at this 
https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/api/EncounterServiceTest.java

Do you suggest this is old so we can clean this out and change the documentation as well.What do you think ?

Reply:
@tendomart i wouldn't suggest to change this documentation in this pr , if we are to change then you can create a separate ticket to address this .
But am kindly suggesting that we don't change the imports order except for the new imports being added 
you can press` ctrl + z` after saving your changes in your IDE to revert these formats

Reply:
You can look at the recommendations here , honestly speaking i though there was no problem with the **import java.util.*;** wild cards  for the java.utils package but it's not an exception either.

but the original code had that , hence the cleaning.

Look at the recommendations at https://wiki.openmrs.org/display/docs/Java+Conventions

Reply:
You can look at the recommendations here , honestly speaking i though there was no problem with the **import java.util.*;** wild cards  for the java.utils package but it's not an exception either.

but the original code had that , hence the cleaning.

Look at the recommendations at https://wiki.openmrs.org/display/docs/Java+Conventions

About the not so obvious code still refer to https://wiki.openmrs.org/display/docs/Java+Conventions for clarification.

Reply:
thanks for your feedback..
am not against the code cleaning and the removing of the wildcards imports but my worry is about the formats as you can see the `org.hamcrest.Matchers `and ` org.junit.jupiter.api` packages moved from the top to the bottom .though we can wait for what others think about.
Thanks .
cc @dkayiwa @ibacher

Reply:
Oh i see ,you mean static imports coming before non static imports ?